exporter/splunkhecexporter/config_test.go

```diff
@@ -101,6 +101,12 @@ func TestLoadConfig(t *testing.T) {
 				},
 				HealthPath:            "/services/collector/health",
 				HecHealthCheckEnabled: false,
+				WorkerQueueConfig: WorkerQueueConfig{
```

Comment:
this config change needs discussion and should be done in a separate PR so we can discuss its merits.